frameworks/g-cloud-9/questions/services/secureDevelopment.yml

```diff
@@ -1,10 +1,9 @@
 name: Approach to secure software development best practice
 question: How does your organisation demonstrate that it adheres to best practice in secure software development?
 question_advice: >
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">CESG CPA Build Standard (new tab)</a>,
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">ISO/IEC 27034 (new tab)</a>,
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">ISO/IEC 27001 (new tab)</a>, or
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">CSA CCM v3.0 (new tab)</a>
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-6-personnel-security" target="_blank"
```

Comment:
Personnel security seems a weird link for this "secure software development" question - is it definitely the one we want?